.github/workflows/test.yml

```diff
@@ -32,6 +33,12 @@ jobs:
           echo "Files to test:"
           echo "$TESTS_ARRAY"
           echo "::set-output name=tests::$TESTS_ARRAY"
+      - uses: patrickedqvist/wait-for-vercel-preview@ae34b392ef30297f2b672f9afb3c329bde9bd487
```

Comment:
Does the `on: deployment_status` workflow trigger not work here? That seems to be how front's playwright setup is triggered.

Reply:
Good point, that would work. However, if we changed to `on: deployment_status` we would have to block for 2 minutes while the deployment is going and wait for the status to be available to GitHub before we could start running turbo to check for the changes. Since both of these take about 2 minutes, it would increase the time before tests beginning running to 4 minutes. However, the way its written in my PR, it will remain 2 minutes.

Reply:
Makes sense, sounds good to kick this one off eagerly instead of waiting for the deployment to start.